Python/bltinmodule.c

```diff
@@ -1623,15 +1623,18 @@ hash as builtin_hash
     obj: object
     /
 
-Return the hash value for the given object.
+Return the integer hash value for the given object within this process.
 
 Two objects that compare equal must also have the same hash value, but the
-reverse is not necessarily true.
+reverse is not necessarily true.  Hash values may vary between Python
+processes.
```

Comment:
Worth mentioning that mutable objects aren't hashable, and that hash() on them raises a TypeError?

Reply:
Something like "Equivalent objects will always give the same hash value within a single Python process, but a different Python process may report a different hash value. Not all objects are hashable."?

Reply:
Or, "The same object can have different hash values in different processes.  Use hash() only within a single process."